packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -50,6 +51,18 @@ const manageSubButtonFn = (onClick: VoidFunction) => {
   );
 };
 
+const iapStopUpgradeSubButtonFn = () => {
+  return (
+    <LinkExternal
+      data-testid="iap-upgrade-get-help-button"
+      className="button primary-button mb-10"
+      href="https://support.mozilla.org/en-US/kb/upgrade-error"
```

Comment:
nit:  drop the `/en-US/` from the path and let SuMo handle l10n negotiation.

Reply:
Updated!